tcpdf.php

```diff
@@ -1274,7 +1274,8 @@ class TCPDF {
 	 * @protected
 	 * @since 4.6.005 (2009-04-24)
 	 */
-	protected $signature_max_length = 11742;
```

Comment:
i''m increasing this size to provide enough space to place tsa response data. and sometimes need more, depend on tsa response data size.